src/plugins/auto/executable_network.cpp

```diff
@@ -239,6 +239,8 @@ MultiDeviceExecutableNetwork::MultiDeviceExecutableNetwork(const std::string&
                       std::call_once(_firstLoadOC, [this] () {
                               _firstLoadPromise.set_value();
                               });
+                      if (_loadContext[CPU].isEnabled && _loadContext[ACTUALDEVICE].isAlready)
```

Comment:
When load network to AUTO with latency hints, AUTO will keep CPU helper thread if CPU is target device.

Reply:
if cpu is target, no recycle thread will be created